cmd/create/cluster/cmd.go

```diff
@@ -46,6 +46,7 @@ import (
 	"github.com/openshift/rosa/pkg/output"
 	"github.com/openshift/rosa/pkg/properties"
 	"github.com/openshift/rosa/pkg/rosa"
+	"k8s.io/apimachinery/pkg/api/resource"
```

Comment:
NP: We should move this import to the middle group of imports, as we tend to group the imports, and keep a whitespace between imports

- std pkgs
- external pkgs
- internal pkgs.